policyengine_us/variables/gov/states/mi/tax/income/credits/homestead_property_tax/mi_homestead_property_tax_credit_household_resource_exemption.py

```diff
@@ -0,0 +1,49 @@
+from policyengine_us.model_api import *
+
+
+class mi_homestead_property_tax_credit_household_resource_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Michigan non-refundable homestead property tax credit"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "http://legislature.mi.gov/doc.aspx?mcl-206-508",
+        "https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2022/2022-IIT-Forms/MI-1040CR.pdf#page=2",
+    )
+    defined_for = StateCode.MI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mi.tax.income.credits.homestead_property_tax
+        p2 = parameters(period).gov.states.mi.tax.income
+        # Line 33
+        total_household_resources = tax_unit("mi_household_resources", period)
+        # Line 5
+        # disabled
+        person = tax_unit.members
+        is_head_or_spouse = person("is_tax_unit_head_or_spouse", period)
+        disabled_person = person("is_disabled", period)
+        disabled_head_or_spouse_present = (
+            tax_unit.sum(disabled_person & is_head_or_spouse) > 0
```

Comment:
Create a new variable in the same general folder space as `is_tax_unit_head_or_spouse` for `disabled_tax_unit_head_or_spouse` and use that in this PR given it uses it multiple times.
```python
            tax_unit.any(disabled_person & is_head_or_spouse)
```